### ISSUE TITLE
Sync error - utils validation issue

### ISSUE THREAD

**Issue by RickDB** [croneter/PlexKodiConnect#997]
## Describe the bug
Upon fresh installation it wouldn't sync and report a .strip() error in logs:

```
21:10:50.142 T:23096   ERROR: PLEX.utils: Error encountered: sync.py crashed - 'NoneType' object has no attribute 'strip'
21:10:50.144 T:23096   ERROR: PLEX.utils: _____________________________________________________________
21:10:50.144 T:23096   ERROR: PLEX.utils:     Traceback (most recent call last):
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\sync.py", line 124, in run
21:10:50.144 T:23096   ERROR: PLEX.utils:         self._run_internal()
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\sync.py", line 200, in _run_internal
21:10:50.144 T:23096   ERROR: PLEX.utils:         self.start_library_sync(block=True)
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\sync.py", line 82, in start_library_sync
21:10:50.144 T:23096   ERROR: PLEX.utils:         library_sync.start(show_dialog, repair, self.on_library_scan_finished)
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\library_sync\full_sync.py", line 461, in start
21:10:50.144 T:23096   ERROR: PLEX.utils:         FullSync(repair, callback, show_dialog).run()
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\library_sync\full_sync.py", line 419, in run
21:10:50.144 T:23096   ERROR: PLEX.utils:         self._run()
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\utils.py", line 1007, in wrapper
21:10:50.144 T:23096   ERROR: PLEX.utils:         result = func(*args, **kwargs)
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\library_sync\full_sync.py", line 428, in _run
21:10:50.144 T:23096   ERROR: PLEX.utils:         if self.isCanceled() or not sections.sync_from_pms(self):
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\library_sync\sections.py", line 590, in sync_from_pms
21:10:50.144 T:23096   ERROR: PLEX.utils:         return _sync_from_pms(pick_libraries)
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\library_sync\sections.py", line 625, in _sync_from_pms
21:10:50.144 T:23096   ERROR: PLEX.utils:         music.excludefromscan_music_folders(sections)
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\music.py", line 43, in excludefromscan_music_folders
21:10:50.144 T:23096   ERROR: PLEX.utils:         parent = xml_file.set_setting(['audio', 'excludefromscan'])
21:10:50.144 T:23096   ERROR: PLEX.utils:       File "C:\Users\Rick\AppData\Roaming\Kodi\addons\plugin.video.plexkodiconnect\resources\lib\utils.py", line 924, in set_setting
21:10:50.144 T:23096   ERROR: PLEX.utils:         old.text.strip() == value and
21:10:50.144 T:23096   ERROR: PLEX.utils:     AttributeError: 'NoneType' object has no attribute 'strip'
```

## To Reproduce
Fresh PKC install / library and sync libraries, oddly enough doesn't happen with all my Plex servers so it can vary between use cases just can't traceback why exactly.

## Possible solution

Additional validation to this part:

https://github.com/croneter/PlexKodiConnect/blob/master/resources/lib/utils.py#L923-L927

As old.text can be NoneType would place the old.text.strip()  in another if clause to prevent this, for instance:

```
if (old is not None):
   if(old.text is not None and old.attrib is not None):
  etc..
```

# Versions

**PKC**: 2.9.5 (latest)
**Kodi**: 17.6

**Comment by croneter** [comment]
Entire debug log please

**Comment by RickDB** [comment]
Here you go:

https://gist.github.com/RickDB/9e128785331179956481072b0bec0011

**Comment by croneter** [comment]
Solved with BETA 2.9.8

**Comment by RickDB** [comment]
Thanks 👍
